octobot_commons/configuration/configuration.py

```diff
@@ -193,6 +193,14 @@ def get_metrics_enabled(self) -> bool:
             )
         )
 
+    def get_metrics_id(self) -> str:
+        """
+        :return: The current user's metrics id
+        """
+        return self.config[commons_constants.CONFIG_METRICS][
```

Comment:
Should we catch a `KeyError` if  `commons_constants.CONFIG_METRICS` doesn't exists?